### PR TITLE
exception 500

### PR DESCRIPTION
**Pull request by fire-poy**
while chatGpt was explain me the code that update photo in user controller, it tell me that a InternalServerErrorException will return a `status code 500`

So I search in the code and replace them for console.logs

Suggestions are welcomed.

### CODE REVIEW

File: backend/src/user/user.controller.ts [hunk 1]
```diff
@@ -137,9 +136,7 @@ export class UserController {
             )
             return { message: 'Profile image saved correctly', photoUrl }
         } catch (error) {
-            throw new InternalServerErrorException(
-                'An error occurred when saving the profile image'
-            )
+            console.log('An error occurred when saving the profile image')
```
Comment (t-h2o) [inline backend/src/user/user.controller.ts:139]: Should we return a 4xx error to inform to the user a bad use ?

**Comment by t-h2o** [comment]
In my opinion, we can leave the error 500.
That make sense, for example:
if we cannot add a new user because the memory is full.

**Comment by RobinBurri** [comment]
I agree with Theo.
Not throwing a 500 where it should, doesn't mean that it's not a 500 error.

**Comment by t-h2o** [comment]
We can abound this idea

**Pull request closed without merging**